content/stable-rc.html

```diff
@@ -7,6 +7,14 @@
 - general
 :author: kohsuke
 ---
+<html>
+<head>
+<meta charset="UTF-8">
+<meta name="description" content="Free Web tutorials">
+<meta name="keywords" content="HTML, CSS, JavaScript">
+<meta name="author" content="John Doe">
```

Comment:
That was just for the owner to get to know about those things that they should be here and so they can fill up in their own more accurately because they know their site better than any other like keywords, description etc.

Reply:
Thanks for the clarification.

Authors and history of recent pages is more readily determined by using the "Improve this page" link at the bottom of each page to open the GitHub source file for the page.  The reader can then press the "Cancel" button to cancel the editing session and can press the "History" link to see the detailed history of the page.  HTML meta tags are not the way that we want to track or record the author of a page.  We already have that information in much greater detail in GitHub. 
 GitHub include who created the change, when did they create that change, what was the pull request that merged it, what were the comments on the pull request, and more.